source/fuzz/fuzzer_pass_replace_copy_objects_with_stores_loads.cpp

```diff
@@ -66,7 +66,15 @@ void FuzzerPassReplaceCopyObjectsWithStoresLoads::Apply() {
                                       ? SpvStorageClassPrivate
                                       : SpvStorageClassFunction;
 
-    // Find or create a constant to initialize the variable from.
+    // Find or create a constant to initialize the variable from. The type of
+    // |instruction| must be scalar or composite.
+    auto instruction_type =
+        GetIRContext()->get_type_mgr()->GetType(instruction->type_id());
+    if (!instruction_type->AsBool() && !instruction_type->AsInteger() &&
+        !instruction_type->AsFloat() &&
+        !fuzzerutil::IsCompositeType(instruction_type)) {
+      return;
+    }
```

Comment:
Is this essentially saying that the type must not be a pointer type? If so, please just check that instead.

Reply:
It is to assert that we can execute FindOrCreateZeroConstant on line 79 so I think that we need to exclude more cases than just pointer types.

Reply:
Ah yes you are right.